src/Core/Domain/Product/Combination/QueryResult/CombinationForEditing.php

```diff
@@ -44,21 +44,21 @@ class CombinationForEditing
     private $prices;
 
     /**
-     * @param CombinationOptions $options
+     * @param CombinationDetails $options
      * @param CombinationPrices $prices
      */
     public function __construct(
-        CombinationOptions $options,
+        CombinationDetails $options,
         CombinationPrices $prices
     ) {
         $this->options = $options;
```

Comment:
Is there a reason for not renaming this `options` parameter and class attribute into `details` ?

Reply:
nope, juts a missed place. 
Updated :+1: